### PR TITLE
Disable zeroconf subscriptions to publishers in the same process

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
@tribal-tec @rdumusc @chevtche @jafyvilla Please review

File: zeq/detail/sender.h [hunk 1]
```diff
@@ -6,8 +6,11 @@
 #ifndef ZEQ_DETAIL_SENDER_H
 #define ZEQ_DETAIL_SENDER_H
 
+#include <zeq/log.h> // ZEQINFO
+#include <zeq/types.h>
 #include <servus/uri.h>
 #include <zmq.h>
+#include <stdexcept>
```
Comment (tribal-tec) [inline zeq/detail/sender.h]: those new includes are not used here?

**Comment by tribal-tec** [comment]
+1

**Comment by favreau** [comment]
Please tell me this is NOT affecting the demo for the summit next week?

**Comment by favreau** [comment]
+1

**Pull request merged**